src/components/Item/CategoryCard.js

```diff
@@ -17,27 +20,41 @@ const useStyles = () => ({
     borderRadius: "100%",
   },
   categoryName: {
-    marginTop: "-1%",
-    fontWeight: "normal",
+    fontWeight: 300,
   },
 });
 
 class CategoryCard extends Component {
+  redirect = async () => {
+    // let category = this.props.value === "all" ? "" : this.props.value;
+    // await this.props.loadPostingsByCategory(category);
```

Comment:
if api call is not made here anymore, could we remove async from the function. Make sure to run yarn format + yarn lint. it'll show you those problems.